applications/openshift/registry/ocp_allowed_registries_for_import/rule.yml

```diff
@@ -7,6 +7,14 @@ description: |-
     image registries from which normal users may import images. This is important
     to control, as a user who can stand up a malicious registry can then import
     content which claims to include the SHAs of legimitate content layers.
+    You can set the allowed repositories for import by applying the following
+    manifest using <code>oc patch</code>:
+    <code>
+    spec:
+      allowedRegistriesForImport:
+      - domainName: my-trusted-registry.internal.example.com
+        insecure: false
+    </code>
```

Comment:
I believe that these tags need to be `<pre/>` instead of `<code/>`. I have seen `<code/>` not render multiple lines of code correctly.

Reply:
Pre is now used